From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: deepak1556 <hop2deep@gmail.com>
Date: Thu, 18 Oct 2018 17:07:17 -0700
Subject: gritsettings_resource_ids.patch

Add electron resources file to the list of resource ids generation.

diff --git a/tools/gritsettings/resource_ids.spec b/tools/gritsettings/resource_ids.spec
index 76a7a799f43b523e12327cc61d5228b84861ed00..6c22b5acb48dc05eb4a84988836c28d0d73f891c 100644
--- a/tools/gritsettings/resource_ids.spec
+++ b/tools/gritsettings/resource_ids.spec
@@ -1252,6 +1252,11 @@
     "includes": [7440],
   },
 
+  "electron/electron_resources.grd": {
+    "messages": [31750],
+    "includes": [31950],
+  },
+
   # END "everything else" section.
   # Everything but chrome/, components/, content/, and ios/
 
